examples/webview-test-app/SpinBox.qml

```diff
@@ -0,0 +1,64 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Item {
+    id: root
+    property alias label: textField.label
+    property real minimumValue: -Infinity
+    property real maximumValue: Infinity
+    property real value: 0.0
+    property real stepSize: 0.1
+
+    implicitHeight: Theme.itemSizeMedium
+    implicitWidth: 200
+
+    onValueChanged: {
+        value = Math.min( Math.max(minimumValue, value), maximumValue )
+    }
```

Comment:
With one liner signal handlers it's not mandatory to use curly brackets. Following would be equally good:

```
onValueChanged: value = Math.min(Math.max(minimumValue, value), maximumValue)